tests/phpunit/Presentation/ListFacetHtmlBuilderTest.php

```diff
@@ -26,30 +25,6 @@ class ListFacetHtmlBuilderTest extends TestCase {
 
 	private const FACET_PROPERTY_ID = 'P42';
 
-	public function testRendersTemplate(): void {
```

Comment:
Why did you remove this test? It still passes with the code change in this PR. Specifically, the StubValueCounter used here still returns items, so the facet should still render.

(The usefulness of this test is a different question, though.)

Reply:
It was removed in an earlier iteration of the PR because it didn't pass, and that the test wasn't useful when we already checks the value of the checkbox in other tests.

Should the test be added back? If so, I'll do it in a follow up PR.